### DISCUSSION TITLE
update pypi libtorrent

### DISCUSSION THREAD

**Discussion by XinfitX** [arvidn/libtorrent#7244] [General]
Hi, thanks very much for all works

https://pypi.org/project/libtorrent/

@AllSeeingEyeTolledEweSew
@arvidn 

is there any update for the pypi packages? is there any solution to the problem?

Thanks very much,

**Comment by ichorid** [comment]
It would be really nice to include support for Python 3.11 too. I guess this is possible now that Boost 1.81 with the relevant patch is out

**Comment by ichorid** [comment]
@AllSeeingEyeTolledEweSew 
@arvidn 

Guys, please? :pray: 
Python 3.11 is taking projects by storm, and it is really a shame that the Libtorrent package is holding them back. It feels so nice when the package is updated regularly :blush:
